HISTORY.rst

```diff
@@ -5,6 +5,8 @@ History
 1.0.1 (2025-05-22)
 ------------------
 
+* Fix double inference when using `predict_set` function in split conformal classification
```

Comment:
Should be above the current version (you can have a look at past versions of this file for an example)